### PR TITLE
Dev.Opera blogpost for Opera 57

### PR DESCRIPTION
**Pull request by fsoder**

### CODE REVIEW

File: src/blog/_posts/2018-11-06-opera-57.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+---
+
+Opera 57 (based on Chromium 70) for [Mac, Windows, Linux](https://www.opera.com/computer) is out! To
+find out what’s new for users, see our
+[Desktop blog post](https://blogs.opera.com/desktop/2018/11/opera-57-stable).
+Here’s what it means for web developers.
+
+## Displaying a dialog will cause pages to leave fullscreen
+
+Dialogs (authentication prompt, payment request et.c) require proper context
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-11-06-opera-57.md]: irregular punctuation for etc
https://data.grammarbook.com/blog/definitions/all-about-etc/

File: src/blog/_posts/2018-11-06-opera-57.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+number of round-trips needed to setup a connection, and remove a number of
+insecure legacy options.
+
+## Layout / CSS
+
+* The `<rp>` element is now `display: none` by default even outside a `<ruby>` element.
+* The behavior of percentage row tracks and gutters in grid containers with
+indefinite heights has been updated, and now behaves the same as (grid) columns.
+
+##  Miscellanea
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-11-06-opera-57.md]: Isn't misc already "plural", so this is many mixes of information?

File: src/blog/_posts/2018-11-06-opera-57.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  within a SourceBuffer. It would for example allow changing the type of media bytes append to a media stream.
+
+## Network
+
+### Intervention Reports
+
+An "intervention" happens when the user agent (the browser) willfully violates
+the request of a web application in order to improve performance, security or
+eliminate behavior that is considered annoying to a user. These interventions
+can now be reported either by using the
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-11-06-opera-57.md]: Would be better to be clear that it's about the web page reporting back to the page author. This is not a feature many web developers are familiar with.

File: src/blog/_posts/2018-11-06-opera-57.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+feature detecting touch support - use `navigator.maxTouchPoints` for
+touchscreen detection and `window.TouchEvent` touch event detection instead. To
+support both touch and mouse input, consider using the [Pointer Events](https://w3c.github.io/pointerevents/) API.
+
+Event listeners for the associated touch events can still be added via
+`EventTarget.prototype.addEventListener`, like:
+
+    document.addEventListener("touchstart", ...);
+
+## Options dictionary for postMessage methods
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-11-06-opera-57.md]: To be consistent postMessage should be highlighted, or Symbol.prototype.description in a header below should not be.

File: src/blog/_posts/2018-11-06-opera-57.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+It was previously possible to access `<frame>` elements by name via their
+parent `<frameset>`. This was a non-standard extension, which has now been
+removed.
+
+## Deprecated features
+
+### Custom Elements "V0" and Shadow DOM "V0"
+
+The "V0" versions of the Custom Elements and Shadow DOM APIs has been
+deprecated, and is expected to be removed in Opera 60 (Chromium 73.) Please
```
Comment (bratell-at-opera) [inline src/blog/_posts/2018-11-06-opera-57.md]: Period outside the parenthesis. 
https://english.stackexchange.com/questions/6632/where-does-the-period-go-when-using-parentheses

File: src/blog/_posts/2018-11-06-opera-57.md (hunk 2 above)
Comment (fsoder) [reply src/blog/_posts/2018-11-06-opera-57.md]: "A collection of diverse items" (Misc is adverb? So would need to be "Miscellaneous items" or so I think.)

File: src/blog/_posts/2018-11-06-opera-57.md (hunk 4 above)
Comment (fsoder) [reply src/blog/_posts/2018-11-06-opera-57.md]: Highlighted (not sure that makes a difference though - style-wise.)

File: src/blog/_posts/2018-11-06-opera-57.md (hunk 5 above)
Comment (fsoder) [reply src/blog/_posts/2018-11-06-opera-57.md]: Fixed.

File: src/blog/_posts/2018-11-06-opera-57.md (hunk 1 above)
Comment (fsoder) [reply src/blog/_posts/2018-11-06-opera-57.md]: Fixed.

File: src/blog/_posts/2018-11-06-opera-57.md (hunk 3 above)
Comment (fsoder) [reply src/blog/_posts/2018-11-06-opera-57.md]: Added "back to the page author" after "reported".

**Pull request merged**